test/functional/fixtures/regression/gh-850/test.js

```diff
@@ -0,0 +1,9 @@
+describe('[Regression](GH-850)', function () {
+    it('Should resume the test if the cursor is moving from reloaded iframe to top window', function () {
+        this.timeout(40000);
```

Comment:
I got timeout error when testing this test in loop locally with Travis. I repeated it and i've got it [this error](http://content.screencast.com/users/GeorgeZA/folders/Jing/media/dcdd2bd5-9ea8-4b43-be68-83f1aebe9884/2016-10-04_1931.png) again (looks like it's better to increase `selectorTimeout` too)